eva-accession-release/src/main/java/uk/ac/ebi/eva/accession/release/batch/io/contig/ContigMongoReader.java

```diff
@@ -105,8 +108,9 @@ private ContigMongoReader(String assemblyAccession, MongoClient mongoClient, Str
         this.aggregation = aggregation;
     }
 
-    private static List<Bson> buildAggregationForActiveContigs(String assemblyAccession) {
-        Bson match = Aggregates.match(Filters.eq(ACTIVE_REFERENCE_ASSEMBLY_FIELD, assemblyAccession));
+    private static List<Bson> buildAggregationForActiveContigs(String assemblyAccession, int taxonomyAccession) {
+        Bson match = Aggregates.match(Filters.and(Filters.eq(ACTIVE_REFERENCE_ASSEMBLY_FIELD, assemblyAccession),
+                Filters.eq(ACTIVE_REFERENCE_TAXONOMY_FIELD, taxonomyAccession)));
```

Comment:
This aggregation applies to the clusteredVariantEntities but the taxonomy field is not really informative there.
The only reliable one is in the submittedVariantEntities.

Reply:
updated